cluster-operator/src/test/java/io/strimzi/operator/cluster/ResourceUtils.java

```diff
@@ -173,6 +173,7 @@ public static Kafka createKafka(String namespace, String name, int replicas,
                         .withLivenessProbe(probe)
                         .withReadinessProbe(probe)
                         .withStorage(new EphemeralStorage())
+                        .withVersion(KafkaVersionTestUtils.LATEST_KAFKA_VERSION)
```

Comment:
Why does this need to be added here? Version is not required field. So if it is required by your changes, you need to fix it.

Reply:
So, do we want to keep this even if it is not needed anymore? I guess it might be useful without it to help with the next change which will miss the point that version is not required ;-)